v2g-liberty/config.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 name: V2G Liberty
-version: 0.4.0
+version: 0.4.2
```

Comment:
Hmm, i expected 0.4.2 in Main as well. If installing from here i get version 0.4.2 anyway.